backend/src/main/java/ai/verta/modeldb/utils/ModelDBUtils.java

```diff
@@ -719,4 +727,56 @@ public static void logAmazonServiceExceptionErrorCodes(Logger LOGGER, AmazonServ
     LOGGER.info("Amazon Service Error Type: " + e.getErrorType());
     LOGGER.info("Amazon Service Error Message: " + e.getErrorMessage());
   }
+
+  public static ResourceVisibility getResourceVisibility(
+      Optional<WorkspaceType> workspaceType, ProtocolMessageEnum visibility) {
+    if (!workspaceType.isPresent()) {
+      return ResourceVisibility.PRIVATE;
+    }
+    if (workspaceType.get() == WorkspaceTypeEnum.WorkspaceType.ORGANIZATION) {
+      if (visibility == ProjectVisibility.ORG_SCOPED_PUBLIC
+          || visibility == RepositoryVisibility.ORG_SCOPED_PUBLIC
+          || visibility == DatasetVisibility.ORG_SCOPED_PUBLIC) {
+        return ResourceVisibility.ORG_CUSTOM;
```

Comment:
But this is for new visibility. if some one use old visibility while creating the project then we should have to convert it into the new one so it will have ORG_CUSTOM as per new one. All other old projects we will migrate from migration batch.

Reply:
Changed